deployment/build_and_release/modules/main.tf

```diff
@@ -58,70 +56,30 @@ resource "google_project_service" "storage_googleapis_com" {
 
 # GCS buckets
 
-# prod log rev 0
-resource "google_storage_bucket" "armored_witness_firmware" {
-  location                    = "EU"
-  name                        = "armored-witness-firmware"
-  storage_class               = "STANDARD"
-  uniform_bucket_level_access = true
-}
-resource "google_storage_bucket" "armored_witness_firmware_log" {
-  location                    = "US"
-  name                        = "armored-witness-firmware-log"
-  storage_class               = "STANDARD"
-  uniform_bucket_level_access = true
+locals {
+  bucket_revisions = ["", "-1"]
```

Comment:
Should we factor out the revisions into the `live` terragrunts above?

I'm thinking about when we might want to roll the log in a given env, it'd be cool to add a new entry to the list of revisions and have the buckets, etc. all created but only in that particular env.

Reply:
That's a good idea. I'm working on another PR right now which splits the `main.tf` so that each env can have its own specific resources. I think it'll be easier to make that change once the structure looks like that, so I'll defer that to a later PR if that works for you?